web/tests/stream_data.test.js

```diff
@@ -944,7 +947,7 @@ test("get_invite_stream_data", () => {
         name: "Inviter",
         stream_id: 25,
         invite_only: true,
-        default_stream: false,
+        subscribed: true,
```

Comment:
Since we are updating the test here, we can test the complete logic like unsubscribed private streams are not included.

Reply:
Done.